src/sequelize_connection_manager.ts

```diff
@@ -52,6 +52,13 @@ export class SequelizeConnectionManager {
       }
     }
 
+    config.retry = {
+      match: [
+         /SQLITE_BUSY/,
+       ],
+      max: 10,
```

Comment:
I am concerned that this may not be enough when we run multiple ProcessInstances for a process with a very large amount of parallel branches.

Let's say we have a Process that uses 10 parallel branches (I believe @cmg-dev 's process has even more than that) and execute that process 2 times. Then this retry-amount would already be at its limit again.

To ensure some type of longevity for this "fix", I'd suggest to crank this up a few notches.

```suggestion
      max: 50,
```